src/api/python/wheels/build_wheel.py

```diff
@@ -0,0 +1,155 @@
+###############################################################################
+# Top contributors (to current version):
+#   Makai Mann
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# ############################################################################
+#
+# Script for building wheels distribution for pycvc5
+#
+# Example usage (from top-level directory):
+#   python3 ./src/api/python/wheels/build_wheel.py bdist_wheel
+# Configures and builds in directory ./build
+# Creates wheel in ./dist
+#
+##
+
+import os
+import re
+import sys
+import platform
+import subprocess
+import multiprocessing
+import shutil
+import glob
+
+from setuptools import setup, Extension
+from setuptools.command.build_ext import build_ext
+from distutils.version import LooseVersion
+
+def get_project_src_path():
+    # expecting this script to be in src/api/python/wheels
+    # The project source directory is 4 directories up
+    # (5 dirnames because start with file)
+    project_src_path = __file__
+    for i in range(5):
+        project_src_path = os.path.dirname(project_src_path)
+    return os.path.abspath(project_src_path)
+
+
+def get_cvc5_version():
+    project_src_path = get_project_src_path()
+
+    # read CMakeLists.txt to get version number
+    version = dict()
+    str_pattern = 'set\(CVC5_(?P<component>MAJOR|MINOR|RELEASE)\s*(?P<version>\d+)\)'
+    pattern = re.compile(str_pattern)
+    with open(project_src_path + '/CMakeLists.txt', 'r') as f:
+        for line in f.read().split('\n'):
+            line = line.strip()
+            m = pattern.search(line)
+            if m:
+                gd = m.groupdict()
+                version[gd['component']] = gd['version']
+                if len(version) == 3:
+                    break
```

Comment:
Instead of reading everything in a string and then splitting it, I'd suggest to either actually read line by line (`for line in f`) or read everything in a string and to `pattern.finditer()` to directly find all matches in the string.

Reply:
Good point! Fixed with 995324034246c5debadd6635f7fbfe5c4b8daaa1.